docs/_snippets/network-response.mdx

```diff
@@ -0,0 +1,23 @@
+<ResponseField name="oid" type="string" required>
+  The OID of this network.
+</ResponseField>
+
+<ResponseField name="name" type="string">
+  The name of this network.
+</ResponseField>
+
+<ResponseField name="zip" type="string">
+  The zipcode this network resides in.
+</ResponseField>
+
+<ResponseField name="state" type="string[]">
+  The two letter statecode for the state this network resides in. For example, `CA`, `NY`, or `TX`.
```

Comment:
Let's try to reuse concept descriptions, e.g.: "The 2 letter state acronym..." (see other .mdx files)

Reply:
addressed